x/distribution/keeper/allocation.go

```diff
@@ -0,0 +1,57 @@
+package keeper
+
+import (
+	"fmt"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/x/distribution/types"
+)
+
+// Allocate fees handles distribution of the collected fees
+func (k Keeper) AllocateFees(ctx sdk.Context) {
+	ctx.Logger().With("module", "x/distribution").Error(fmt.Sprintf("allocation height: %v", ctx.BlockHeight()))
+
+	// if there is no power in the system nothing should be allocated
+	bondedTokens := k.stakeKeeper.TotalPower(ctx)
```

Comment:
Huh when would we hit this case - how would a block be produced with zero power?

Reply:
This is an edge case which shouldn't apply in the working hub, however I can see alternate systems where this may be appropriate (total power could be determined independent of Tendermint validator power in other systems). No reason this should lead to a panic IMO so just skip allocation if no power during allocation process